packages/frontend/editor-ui/src/features/settings/usage/views/SettingsUsageAndPlan.vue

```diff
@@ -321,7 +321,7 @@ const openCommunityRegisterModal = () => {
 					<N8nButton type="secondary" @click="activationKeyModal = false">
 						{{ locale.baseText('settings.usageAndPlan.dialog.activation.cancel') }}
 					</N8nButton>
-					<N8nButton :disabled="!activationKey" @click="onLicenseActivation">
+					<N8nButton :disabled="!activationKey" @click="() => onLicenseActivation()">
```

Comment:
Ooh good catch. Without wrapping it in the lambda the click event will be passed into `onLicenseActivation`.